*** wine_original/dlls/wined3d/arb_program_shader.c	2011-06-18 23:24:08.500021110 +0200
--- wine-1.3.22/dlls/wined3d/arb_program_shader.c	2011-06-18 23:24:47.078406076 +0200
*************** static HRESULT arbfp_blit_set(void *blit
*** 7067,7080 ****
  
-     if (!is_complex_fixup(surface->resource.format->color_fixup))
-     {
-         TRACE("Fixup:\n");
-         dump_color_fixup_desc(surface->resource.format->color_fixup);
-         /* Don't bother setting up a shader for unconverted formats */
-         ENTER_GL();
-         glEnable(textype);
-         checkGLcall("glEnable(textype)");
-         LEAVE_GL();
-         return WINED3D_OK;
-     }
- 
      fixup = get_complex_fixup(surface->resource.format->color_fixup);
--- 7067,7068 ----
